mlflow/pyfunc/__init__.py

```diff
@@ -2058,6 +2089,7 @@ def save_model(
     metadata=None,
     model_config=None,
     example_no_conversion=False,
+    streamable=False,
```

Comment:
DO we need this flag? Can we inspect the model instance to check if it's streamable or not?

Reply:
For this case `_save_model_with_loader_module_and_data_path`, we can't inspect `streamable` .

I can make `streamable` default value to None. so for `mlflow.pyfunc.model._save_model_with_class_artifacts_params`, it can auto inspect `streamable` if it is set to None, for `_save_model_with_loader_module_and_data_path` case, None means streamable=False, thoughts ?

Reply:
> I can make streamable default value to None

sounds good